client/scss/components/_listing.scss

```diff
@@ -91,7 +91,10 @@ ul.listing {
     }
   }
 
-  &:has(.bulk-actions-filter-checkbox) {
+  &:has(
+      td:first-child input[type='checkbox'],
+      th:first-child input[type='checkbox']
```

Comment:
The form submissions checkboxes don't use the bulk actions mechanism, so it doesn't have `bulk-actions-filter-checkbox`. I changed the logic to check whether the first cell has a checkbox element instead.

Seems to work fine, I tested this with:

- new universal listings
  - pages (with bulk actions vs with custom ordering active)
  - snippets, images, documents (which have bulk actions)
  - `ModelViewSet` (no bulk actions)
- old listing without bulk actions (workflows, groups, etc.)
- old listing with bulk actions (users)
- chooser listing with no checkboxes
- chooser listing with checkboxes (`MultipleChooserPanel`)